lib/router/with-router.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import hoistStatics from 'hoist-non-react-statics'
+
+export default function withRouter (options = {}) {
+  const { routerPropName = 'router' } = options
+
+  return function withRouterComponent (ComposedComponent) {
+    const displayName = ComposedComponent.displayName || ComposedComponent.name || 'Unknown'
```

Comment:
@HaNdTriX This PR is into the `master` branch (per instructions from @arunoda), and so far only `v3-beta` has the `getDisplayName` util.

Should I change this PR to be for `v3-beta` instead? Is it possible to change the requested branch for a PR that's already been created?